docs/python-quickstart.md

```diff
@@ -26,45 +26,431 @@ menu:
+                      ├── 12: extra: optional double                                                                                                                                                                    
+                      ├── 13: mta_tax: optional double                                                                                                                                                                  
+                      ├── 14: tip_amount: optional double                                                                                                                                                               
+                      ├── 15: tolls_amount: optional double                                                                                                                                                             
+                      ├── 16: improvement_surcharge: optional double                                                                                                                                                    
+                      ├── 17: total_amount: optional double                                                                                                                                                             
+                      ├── 18: congestion_surcharge: optional double                                                                                                                                                     
+                      └── 19: airport_fee: optional double                                                                                                                                                              
+Current snapshot      Operation.APPEND: id=5937117119577207079, schema_id=0                                                                                                                                             
+Snapshots             Snapshots                                                                                                                                                                                         
+                      └── Snapshot 5937117119577207079, schema 0: file:/.../nyc.db/taxis/metadata/snap-5937117119577207079-1-94656c4f-4c66-4600-a4ca-f30377300527.avro
+Properties            owner                 root                                                                                                                                                                        
+                      write.format.default  parquet
 ```
-# simply run tox from within the python dir
-tox
+
+Or output in JSON for automation:
+
+```sh
+pyiceberg --uri thrift://localhost:9083 --output json describe nyc.taxis | jq
+{
+  "identifier": [
```

Comment:
This is a bit long. Could we snip some parts of the schemas?

Reply:
Agreed, pruned the schemas